guardian/prosemirror-elements
# Namespace field nodes with their element type, and add tests

## What does this change?

Namespaces nodes representing fields with their element type. This prevents different elements with identical field names conflicting, which is currently blocking #65. 

## How to test

- The automated tests should pass.
- Launch the demo and examine the Prosemirror schema in the devtools. Each node type should be clearly namespaced with the field name:

![Screenshot 2021-08-03 at 13 20 00](https://user-images.githubusercontent.com/7767575/128014378-da7faf9d-6c64-47c3-92d0-59dcffa91664.png)
